### PR TITLE
Calibration fox changes

### PR DESCRIPTION
**Pull request by Louka3000**
- Changed manual configuration of body proportions.
- Added instructions on which SteamVR to enable.
- Copied straight from the discord #calibration channel.

### CODE REVIEW

File: slimevr-setup.md [hunk 1]
```diff
@@ -122,6 +122,20 @@ Before launching SteamVR, check the checkboxes relative to how many SteamVR virt
...
 
+### SteamVR trackers to enable depending on your set:
+
+5+0: waist and legs.
+5+1: *chest*, waist and legs.
+5+3: *chest*, waist, *knees*, legs.
+
+Trackers marked in italic should only be enabled if your game supports it.
+
+*Knee trackers should only be enabled when you have feet extensions.*
```
Comment (Eirenliel) [inline slimevr-setup.md:133]: Why? They would work either way.

File: slimevr-setup.md [hunk 2]
```diff
@@ -122,6 +122,20 @@ Before launching SteamVR, check the checkboxes relative to how many SteamVR virt
...
+
+*(You may notice a more precise upper body (chest and arms) and spine but a less precise lower body (hip and legs) when enabling chest tracker depending on your avatar.)*
+
+Do not enable trackers you do not need as this can lead to calibration issues in-game.
```
Comment (Eirenliel) [inline slimevr-setup.md:137]: Need to add note that some games, like unmodded VRChat don't support more than 3 trackers, so anything beyond waist and legs shouldn't be enabled even when using bigger sets.

**Review by Eirenliel — CHANGES_REQUESTED** [review]

**Comment by Louka3000** [comment]
> Need to add note that some games, like unmodded VRChat don't support more than 3 trackers, so anything beyond waist and legs shouldn't be enabled even when using bigger sets.
Added a commit in my PR fixing this.
`Do not enable trackers you do not need as this can lead to calibration issues in-game (this includes chest and knee trackers for VRChat because it only supports waist and legs trackers!).`

**Comment by calliePepper** [comment]
I do like the addition of the range for each value, that's a reallllly good call

**Pull request merged**